atm.js

```diff
@@ -1,3 +1,118 @@
 $(document).ready(function(){
-// Uh oh -- it's saying `$` is undefined! Something's missing from `index.html`...
+
+     console.log("Hola hola hola");
+
+// CHECKING DEPOSIT
+
+$("#checking > .deposit").on("click", function(evt){
+    // Prevent default action
+    evt.preventDefault();
+
```

Comment:
Nice job pseudocoding! You actually don't even need the > for the selector here because there's only one deposit class inside checking. I'd only use the > id if you were worried about capturing other descendants that weren't direct children, but you don't have that here.